zkvm/src/ops.rs

```diff
@@ -277,3 +282,90 @@ impl Instruction {
         }
     }
 }
+
+macro_rules! def_op {
+    ($func_name:ident, $op:ident) => (
+           pub fn $func_name(&mut self) -> &mut Program{
+             self.0.push(Instruction::$op);
+             self
+        }
+    );
+}
+
+impl Program {
+    def_op!(drop, Drop);
```

Comment:
Maybe for clarity we can order all ops in the canonical order, and have an extra macro `def_op_usize` for dup/roll/output/contract?